CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), 
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.2] - 2022-01-06
+
+### Added
+
+* Metadata for the Dividend, Property and Trading Allowances
```

Comment:
Per keepachangelog standard. Could you also add to 0.9.1?
```suggestion
* Metadata for the Dividend, Property and Trading Allowances.
```

Reply:
Added!